### ISSUE TITLE
quasar v0.9.67 linux armv7, cant find steam, armbian orangepi pc

### ISSUE THREAD

**Issue by ksalaz** [scakemyer/plugin.video.quasar#910]
Hello!
please help me with such problem. i use orange pi pc and armbian v5.36. i install kodi 15.2 from repository and install quasar v0.9.67 linux armv7. after i install quasar burst from quasar repository, and try search something. i got error "cant find stream", but in log file streamin are found.
this is my log file
https://pastebin.com/sxTmNRc4
whot is this problem? can anyone help me?

**Comment by tomtomclub** [comment]
hey try 4 provider's  no more

**Comment by dordort** [comment]
Why? just use the "Burst" addon.

2017-12-26 22:54 GMT+02:00 tomtom <notifications@github.com>:

**Comment by ksalaz** [comment]
yes, i disable providers who wrote "working slow" and use only english words in search and its works ).
1080p working slow, 720p is good )
